hack/test-cmd.sh

```diff
@@ -267,6 +268,28 @@ runTests() {
   ! kubectl create
 
   #######################
+  # kubectl config set #
+  #######################
+
+  kube::log::status "Testing kubectl(${version}:config set)"
+
+  kubectl config set-cluster test-cluster --server="https://127.0.0.1:${TLS_API_PORT}"
```

Comment:
Is there any way to get the value of the ip address that kube is using here. localhost and 127.0.0.1 aren't in the autogenerated certs subject-alt-names. I guess the alternative is to skip the check that this actually works. ```kubectl api-versions --cluster=test-cluster --username=none```.

Changing the service-cluster-ip-range breaks other tests.